client/src/hooks/useDownloadModal.js

```diff
@@ -18,22 +18,23 @@ export const useDownloadModal = (
     initialPublicComputedFile
   )
   const [download, setDownload] = useState(false)
-
   const hasMultipleFiles = hasMultiple(resource.computed_files)
-
   // states that dictate what the modal can show
   const isDownloadReady = download && token
-  const isTokenReady = !token && publicComputedFile
   const isOptionsReady = !publicComputedFile && hasMultipleFiles
-
+  const isSampleMetadataOnly = publicComputedFile?.metadata_only
+  const isTokenReady = !token && publicComputedFile
   // text information
   const verb = isDownloadReady ? 'Downloading' : 'Download'
-  const resourceType = resource.samples ? 'Project' : 'Sample'
-  const modalTitle = `${verb} ${resourceType}`
-  const buttonLabel = `Download ${resourceType}`
-
+  // eslint-disable-next-line no-nested-ternary
+  const resourceType = isSampleMetadataOnly
+    ? 'Sample'
+    : resource.samples
+    ? 'Project'
+    : 'Sample'
+  const metadata = isSampleMetadataOnly ? 'Metadata' : ''
+  const modalTitle = `${verb} ${resourceType} ${metadata}`
```

Comment:
```suggestion
  const resourceType = resource.samples
    ? 'Project'
    : 'Sample'
  const modalTitle = isSampleMetadataOnly
    ? `${verb} Sample Metadata`
    : `${verb} ${resourceType}`
```

I think breaking it up this way is a bit easier to follow.

Reply:
See the commit at c080be77ed9adface1fdaa0b722f365ee8ab6f4e.